app/webpacker/packs/school_search.js

```diff
@@ -1,5 +1,5 @@
 import accessibleAutocomplete from "accessible-autocomplete";
-import fetch from "node-fetch";
+import "whatwg-fetch";
```

Comment:
it's a very minor thing, but this would be best in application.js as probably won't be specific to this file for very long if any other JS functionality is added

Reply:
Done